.github/workflows/test-playbooks.yml

```diff
@@ -0,0 +1,62 @@
+name: Test Ansible Playbooks
+
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - "**.md"
+      - "**.example"
+  pull_request:
+    branches:
+      - master
+    paths-ignore:
+      - "**.md"
+      - "**.example"
+
+jobs:
+
+  build-docker-image:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [centos-7, centos-8, ubuntu-18.04, ubuntu-20.04]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build the Docker image
+        run: docker build . --file ./.github/workflows/Dockerfile-${{ matrix.os }} --tag "docker.pkg.github.com/$(echo $GITHUB_REPOSITORY | tr '[A-Z]' '[a-z]')/${{ matrix.os }}"
+      - name: Docker login
+        run: docker login docker.pkg.github.com -u $GITHUB_ACTOR -p $GITHUB_TOKEN
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Publish to GPR
+        run: docker push "docker.pkg.github.com/$(echo $GITHUB_REPOSITORY | tr '[A-Z]' '[a-z]')/${{ matrix.os }}"
```

Comment:
This should be conditional, skip when the workflow is being run for a PR?

Reply:
Do we actually *need* to this?

Just `docker build` and then `docker run` should work?

And only if that works out, `docker push` for non-PR runs?

Reply:
I think I tried that, but then the `docker run` complained that the image didn't exist. Maybe because they were running as different steps? I can try again, and put both commands in the same step.

Reply:
Hmm, don't know why that didn't work, it does work now (even as separate steps). Do we still want to keep the image? Don't think we really need it.

Reply:
If we don't need to image afterwards, there's no point in storing it, indeed. And it simplifies the workflow.